client/src/sockets/enums.js

```diff
@@ -35,6 +35,7 @@ export const ListEvents = Object.freeze({
   ADD_VIEWER_SUCCESS: ListActionTypes.ADD_VIEWER_SUCCESS,
   ARCHIVE_SUCCESS: ListActionTypes.ARCHIVE_SUCCESS,
   CHANGE_TYPE_SUCCESS: ListActionTypes.CHANGE_TYPE_SUCCESS,
+  DELETE_AND_REDIRECT: 'list/DELETE_AND_REDIRECT',
```

Comment:
why not in `ListActionTypes`?

Reply:
beacuse this is specific to socket and we don't need action type for this.
The same approach is in the line 43 and 45 in this file. 
Check this out